projects/golang/go/build/conformanceTests/golangBuildspec.yaml

```diff
@@ -0,0 +1,66 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#      http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file was generated by running `make generate`. DO NOT EDIT
+
+version: 0.2
+batch:
+  fast-fail: true
+  build-graph:
+    - identifier: golang_115
+      env:
+        variables:
+          GO_SOURCE_VERSION: 1.15
+    - identifier: golang_116
+      env:
+        variables:
+          GO_SOURCE_VERSION: 1.16
+    - identifier: golang_117
+      env:
+        variables:
+          GO_SOURCE_VERSION: 1.17
+    - identifier: golang_118
+      env:
+        variables:
+          GO_SOURCE_VERSION: 1.18
+    - identifier: golang_119
+      env:
+        variables:
+          GO_SOURCE_VERSION: 1.19
+    - identifier: golang_120
+      env:
+        variables:
+          GO_SOURCE_VERSION: 1.20
```

Comment:
Once this merges we should add this to the runbook as another place to update when making changes/adding a new minor version.